ui/tabs.js

```diff
@@ -664,24 +669,27 @@ return $.widget( "ui.tabs", {
 			"aria-selected": "false",
 			"aria-expanded": "false"
 		} );
+
 		// If we're switching tabs, remove the old tab from the tab order.
 		// If we're opening from collapsed state, remove the previous tab from the tab order.
 		// If we're collapsing, then keep the collapsing tab in the tab order.
 		if ( toShow.length && toHide.length ) {
 			eventData.oldTab.attr( "tabIndex", -1 );
 		} else if ( toShow.length ) {
-			this.tabs.filter( function() {
-				return $( this ).attr( "tabIndex" ) === 0;
-			} )
-				.attr( "tabIndex", -1 );
+			this.tabs
+				.filter( function() {
+					return $( this ).attr( "tabIndex" ) === 0;
+				} )
+					.attr( "tabIndex", -1 );
```

Comment:
This is effectively the same case as what I mentioned earlier. Since the first operation is a context change, we should be able to keep it on one line, which gets us to:

```js
this.tabs.filter( function() {
    return $( this ).attr( "tabIndex" ) === 0;
} )
    .attr( "tabIndex", -1 );
```